extensions/eclipse-che-theia-plugin-remote/tslint.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
You do not need this file anymore, you use `tslint.json` from 'configs/tslint.json'